packages/admin/admin/src/FinalFormSaveSplitButton.tsx

```diff
@@ -11,9 +11,10 @@ import { useStackApi } from "./stack/Api";
 
 export interface FormSaveButtonProps {
     localStorageKey?: string;
+    hasConflict?: boolean;
```

Comment:
I don't really like this. Of course, Final Form has no concept of conflicts in it's form state. Maybe we could add a mutator to add the conflict state to Final Form? https://final-form.org/docs/final-form/types/Mutator

@nsams what do you think?

Reply:
You are right, it is strange that `FinalFormSaveSplitButton` gets it's data internally by accessing the from api thru context, except conflicts.

Yes, we could add the conflict state to the form; we could even add the whole conflict management into the form (instead of `useFormSaveConflict`).

However I always thought is is good to not have yet another feature added to final-form when it is also possible to implement from the "outside".

Reply:
Fine by me. @nsams what do you think?